net.gaijin.WarThunder.desktop

```diff
@@ -0,0 +1,13 @@
+[Desktop Entry]
+Version=1.0
+Name=War Thunder
+Exec=run.sh
+Terminal=false
+Type=Application
+Icon=net.gaijin.WarThunder
+Categories=GNOME;GTK;Game;
```

Comment:
`GNOME;GTK`?

Reply:
It is how the game itself Categorizes itself.

These do not throw errors, it seems to be valid.

Reply:
This is less about validity and more about relevancy. It doesn't make any sense to have it marked as GNOME/GTK.